cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/data_explorer/data_source_selector.spec.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  MiscUtils,
+  TestFixtureHandler,
+} from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+const testFixtureHandler = new TestFixtureHandler(
+  cy,
+  Cypress.env('openSearchUrl')
+);
+
+describe('Data source selector', () => {
+  before(() => {
+    testFixtureHandler.importJSONMapping(
```

Comment:
Haven't merged the `main` pr yet but could you add the setting of the global tenant?

https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/963/files. Without the security plugin this is fine but once the security plugin is added the default tenant is the private tenant and not the `.kibana`